test/new_relic/rack/browser_monitoring_test.rb

```diff
@@ -222,6 +222,32 @@ def test_content_length_set_when_response_is_nil
       assert_equal '0', headers['Content-Length']
     end
 
+    # introduce an exception in the middle of #traced_call and verify that
```

Comment:
For documentation's sake, what do you think about adding a test with a Content-Type header that's a symbol?

Reply:
Added with [5ef2fbd](https://github.com/newrelic/newrelic-ruby-agent/pull/2465/commits/5ef2fbde9850476c902e7106df51de8acb041be9)